### ISSUE TITLE
Incorrect line numbers after multi line string

### ISSUE THREAD

**Issue by paran1** [rodjek/puppet-lint#565]
inline_template with multiple lines cause puppet-lint to report incorrect line number for subsequent errors in the same file. Tested using puppet-lint 2.0.2.

Example without problem. Class `undocumented` on line 6 is reported as missing documentation.
```
$ nl inline_template.pp
     1	# foo
     2	class foo
     3	{
     4	  $foo = inline_template('<%=foo.foo-%>')
     5	}
     6	class undocumented {}
$ puppet-lint --with-context --only-checks documentation inline_template.pp
WARNING: class not documented on line 6

  class undocumented {}
  ^

$
```

And now the bad behaviour. The inline_template now have two lines, moving the `undocumented` class to line 7, but it gets reported on line 6.
```
$ nl multiline_inline_template.pp
     1	# foo
     2	class foo
     3	{
     4	  $foo = inline_template('<%=foo.
     5	    foo-%>')
     6	}
     7	class undocumented {}
$ puppet-lint --with-context --only-checks documentation multiline_inline_template.pp
WARNING: class not documented on line 6

  }
  ^

$ 
```

**Comment by paran1** [comment]
This was caused by the multi line string itself, and had nothing to do with inline_template... (I have updated the issue title)
This was mostly fixed by the recently merged pull request #570. However that only fixed for single quoted strings, I have created pull request #577 with a trivial patch that uses the same code for double quoted strings.

**Comment by paran1** [comment]
And I missed #403, this one is a duplicate.

**Comment by rnelson0** [comment]
Closing as a duplicate then, thanks!
